firefox-ios/Client/Frontend/MicroSurvey/MicroSurveyPrompt.swift

```diff
@@ -0,0 +1,135 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import ComponentLibrary
+
+struct MicroSurveyViewModel {
+    // TODO: FXIOS-8987 - Add Strings
+    var title: String = "Help make printing in Firefox better. It only takes a sec."
+    var buttonText: String = "Continue"
```

Comment:
Q. Did you mean that these will get added to actual strings file?

Reply:
If so we can just leave it empty instead of adding full string values or mark it commented to avoid confusion